packages/webdriverio/tests/commands/browser/execute.test.ts

```diff
@@ -11,15 +11,27 @@ describe('isEnabled test', () => {
             }
         })
 
-        await browser.execute(() => 'foobar', 1, 2, 3)
+        await browser.execute((a, b, c) => a + b + c, 1, 2, 3)
```

Comment:
Because of more strict definition, typescript now complains about undefined arguments.